packages/schema/src/config/vite.ts

```diff
@@ -56,13 +56,15 @@ export default defineUntypedSchema({
         },
       },
       script: {
-        propsDestructure: {
-          $resolve: async (val, get) => val ?? Boolean((await get('vue') as Record<string, any>).propsDestructure),
-        },
         hoistStatic: {
           $resolve: async (val, get) => val ?? (await get('vue') as Record<string, any>).compilerOptions?.hoistStatic,
         },
       },
+      features: {
+        propsDestructure: {
+          $resolve: async (val, get) => val ?? Boolean((await get('vue') as Record<string, any>).propsDestructure),
```

Comment:
we should probably also check if `vue.script.propsDestructure` is set, for backwards-compatibility